projects/Mallard/src/hooks/use-image-paths.ts

```diff
@@ -63,7 +63,7 @@ export const useImagePath = (image?: Image, use: ImageUse = 'full-size') => {
     const { issueId } = useIssueSummary()
 
     const [paths, setPaths] = useState<string | undefined>()
-    const apiUrl = useSettingsValue.apiUrl()
+    const apiUrl = useApiUrl() || ''
```

Comment:
This might be fine (it might have been what we were doing before and is handled elsewhere) but an empty string for an API url doesn't seem like it would yield the right behaviour?

Reply:
> it might have been what we were doing before

Yup, the existing code is broken afaik, because it wasn't handling the loading state despite the fact there was one. This is because `apiUrl` was typed as `any`. This surfaces now because we have a strongly typed `useApiUrl` function. We should look into fixing this, however I'd recommend that we do that as a second step. Normally, since this would reload with the correct value eventually, we'd have the right final value.

Reply:
Can we added a dreaded `TODO` here then ... at least to mention it as an issue.

Reply:
Will add